cross/tt-rss/Makefile

```diff
@@ -1,9 +1,9 @@
 PKG_NAME = tt-rss
-PKG_VERS = 328d7b55c8
+PKG_VERS = 473ea6255c
 PKG_EXT = tar.gz
 PKG_DOWNLOAD_METHOD = git
-PKG_GIT_HASH = 328d7b55c8cb4d56492356a73df19f7c59784d82
-PKG_DIST_SITE = https://git.tt-rss.org/fox/tt-rss.git
+PKG_GIT_HASH = 473ea6255c634fa47ee8c8e24de910785b9b095b
+PKG_DIST_SITE = https://github.com/smaarn/tt-rss.git
```

Comment:
Ok this is bold, I give you that.
But at some point we will need to make an "arbitrage" on this package since it seems that official repository does not take it very well when multiple clones are being done simultaneously.